src/DropdownItem.js

```diff
@@ -0,0 +1,158 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import all from 'prop-types-extra/lib/all';
+import elementType from 'prop-types-extra/lib/elementType';
+
+import chain from './utils/createChainedFunction';
+import SafeAnchor from './SafeAnchor';
+import SelectableContext, { makeEventKey } from './SelectableContext';
+import { createBootstrapComponent } from './ThemeProvider';
+import mapContextToProps from './utils/mapContextToProps';
+import NavContext from './NavContext';
+
+class DropdownItem extends React.Component {
+  static propTypes = {
+    /** @default 'dropdown' */
+    bsPrefix: PropTypes.string.isRequired,
+
+    /**
+     * Highlight the menu item as active.
+     */
+    active: PropTypes.bool,
+
+    /**
+     * Disable the menu item, making it unselectable.
+     */
+    disabled: PropTypes.bool,
+
+    /**
+     * Styles the menu item as a horizontal rule, providing visual separation between
+     * groups of menu items.
+     */
+    divider: all(
```

Comment:
we should probably change this API for dividers

Reply:
Cuz it's weird?

Reply:
yeah, we were pretty confused by this API :p and couldn't use it for our own dropdown menu dividers!